kernel/arch/dreamcast/kernel/cache.s

```diff
@@ -201,6 +207,7 @@ _dcache_purge_all:
 ! r4 is address for temporary buffer 32-byte aligned
 ! r5 is size of temporary buffer (8 KB or 16 KB)
 _dcache_purge_all_with_buffer:
+.align 2
```

Comment:
This gives improvements for all functions?

Reply:
For the most part yes,  some are very minor ~1000ns

Reply:
Shouldn't the alignment directives be before the symbols, not after?

Reply:
I dont think it matters but I can move them if that is the proper way to do it.  Im still an asm noob and Ive seen so many approaches.  Let me look it up right now.

Reply:
@ljsebald You're right.  I'll make the changes.